core/src/integration-test/java/io/aiven/kafka/tieredstorage/RemoteStorageManagerTest.java

```diff
@@ -465,6 +469,79 @@ void testRequiresCompression(final CompressionType compressionType, final boolea
         assertThat(requires).isEqualTo(expectedResult);
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    void testTransformingIndexes(final boolean encryption) {
```

Comment:
I'd say these tests should not be in this class but somewhere in `/test` directory since these are plain unit tests.